source/driver/Driver.cpp

```diff
@@ -612,6 +612,12 @@ bool Driver::parseAllSources() {
                         missing.emplace(name);
                 }
             }
+
+            for (auto bd : meta.interfacePorts) {
+                std::string_view name = bd->nameOrKeyword.valueText();
+                if (knownNames.find(name) == knownNames.end())
```

Comment:
I took out the interface keyword check bc of the Parser change. I don't think the name will every be empty right?

Reply:
I think you still need those checks because of the reconstruction case in ParserMetadata.cpp, which will apply to all InterfacePortHeaders and not just the specific code path in parsePortHeader. That check can either be here or in ParserMetadata.cpp.